db/schema.rb

```diff
@@ -414,7 +415,8 @@
     t.string "date_format", default: "%m/%d/%Y"
     t.string "signage_convention", default: "inflows_positive"
     t.string "error"
-    t.string "number_format"
+    t.string "number_format", default: "1,234.56"
```

Comment:
Migrations are looking good now, but it appears that this is still from a stale migration.  Running `bin/rails db:migrate:reset` should fix and remove this.  If it doesn't let me know.

As you can see in production: 

https://github.com/maybe-finance/maybe/blob/8539ac7dec4c713fc6f3b92abf05108a261bd194/db/schema.rb#L417

This default doesn't exist.  And we didn't alter this at all in this PR, so it shouldn't be showing up in the output schema.